fields/primitives_test.go

```diff
@@ -44,4 +44,10 @@ func TestTextMarshalQualifiedHash(t *testing.T) {
 	if !q.Equals(out) {
 		t.Fatalf("Input and output do not match:\nin: %v\nout: %v", *q, *out)
 	}
+
+	q.Blob = []byte(`this
```

Comment:
I love everything about this PR except that this check is in the wrong place. We're realistically concerned about `QualifiedContent` types containing newlines (or not) since those types are used for Arbor usernames. If you wanted to test it on a qualified type, Content seems like a weird choice. You could also just make a separate `TestBlobContains` that looked kinda like:
```go
func TestBlobContains(t *testing.T) {
b := fields.Blob([]byte("whatever"))
if b.ContainsString("w") {
// yada yada
```

Reply:
Yea, I did put that test in the wrong place. I'll move it.

Reply:
Moved the test.